components/BottomNavigation/src/MDCBottomNavigationBarController.m

```diff
@@ -195,6 +260,64 @@ - (void)didUpdateNavigationBarItemsWithNewValue:(NSArray *)items {
   }
 }
 
+#pragma mark - Touch Events
+
+/** Handles long press gesture recognizer event updates. */
+- (void)handleNavigationBarLongPress:(UIGestureRecognizer *)recognizer {
+  CGPoint touchPoint = [recognizer locationInView:self.navigationBar];
+  switch (recognizer.state) {
+    case UIGestureRecognizerStateBegan:
+    case UIGestureRecognizerStateChanged:
+      [self handleNavigationBarLongPressUpdatedForPoint:touchPoint];
+      break;
+    default:
+      [self handleNavigationBarLongPressEndedForPoint:touchPoint];
+      break;
+  }
+}
+
+/**
+ * Handles when the navigation bar long press gesture recognizer gesture has been initiated or the
+ * touch point was updated.
+ * @param point CGPoint The point within @c navigationBar coordinate space.
+ */
+- (void)handleNavigationBarLongPressUpdatedForPoint:(CGPoint)point {
+  UIFont *preferredFont = [UIFont preferredFontForTextStyle:UIFontTextStyleBody];
```

Comment:
Based on the design document I would expect to check the `UIContentSizeCategory` value.  Is there a reason why it wouldn't be possible and we need to check a font? 

For this controller, we should probably read it using the following logic:

```objc
- (BOOL)contentSizeCategoryIsAccessibilityCategory {
  UIContentSizeCategory sizeCategory = UIContentSizeCategoryLarge;
  if (@available(iOS 10.0, *)) {
    sizeCategory = self.traitCollection.preferredContentSizeCategory;
  } else {
    sizeCategory = UIApplication.mdc_safeSharedApplication.preferredContentSizeCategory;
  }
  if (@available(iOS 11.0, *)) {
    return UIContentSizeCategoryIsAccessibilityCategory(sizeCategory);
  }
  if ([sizeCategory isEqualToString:UIContentSizeCategoryAccessibilityMedium] ||
      [sizeCategory isEqualToString:UIContentSizeCategoryAccessibilityLarge] ||
      [sizeCategory isEqualToString:UIContentSizeCategoryAccessibilityExtraLarge] ||
      [sizeCategory isEqualToString:UIContentSizeCategoryAccessibilityExtraExtraLarge] ||
      [sizeCategory isEqualToString:UIContentSizeCategoryAccessibilityExtraExtraExtraLarge]) {
    return YES;
  }
  return NO;
}
```

Reply:
Done.